src/AutoRest.CSharp/Common/Generation/Writers/CodeWriterExtensions.cs

```diff
@@ -31,6 +31,41 @@ public static CodeWriter AppendNullableValue(this CodeWriter writer, CSharpType
             return writer;
         }
 
+        public static IDisposable Class(this CodeWriter writer, string className, string accessibility = "public", bool isStatic = false)
```

Comment:
We tried helpers like these in the beginning and decided to avoid them. The call to the helper isn't much shorter than the `writer.Scope` but you end up with a helper method that has a bazillion parameter.

Reply:
Fixed